crates/cuid1/Cargo.toml

```diff
@@ -24,6 +25,11 @@ once_cell = "1.9.0"
 rand = "~0.8.0"
 sha3 = "0.10.8"
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+getrandom = { version = "0.2.11", features = ["js"] }
```

Comment:
`getrandom` is not explicitly used, but this line is needed for feature unification, correct?

Reply:
`rand` crate depends on `getrandom`, but does not enable js feature explicitly.